framework/status/__init__.py

```diff
@@ -26,10 +26,20 @@ def push_status_message(message, kind='warning', dismissible=True, trust=True, j
     :param dismissible: Whether the status message can be dismissed by the user
     :param trust: Whether the text is safe to insert directly into HTML as given. (useful if the message includes
         custom code, eg links) If false, the message will be automatically escaped as an HTML-safe string.
-    :param jumbotron: Should this be in a jumbstron element rather than an alert
+    :param jumbotron: Should this be in a jumbotron element rather than an alert
     """
     # TODO: Change the default to trust=False once conversion to markupsafe rendering is complete
-    statuses = session.data.get('status')
+    try:
+        statuses = session.data.get('status')
+    except RuntimeError:
```

Comment:
My only concern is that this _might_ be a bit too greedy with error handling. Is it possible for this to raise a `RuntimeError` anywhere other than what this is intended to fix, and if so, do we want to prevent failure there? If the answer to either of those is `Well, maybe...` or stronger, is there a way of verifying that *this* particular `RuntimeError` happened because of an API call?